devcluster/devcluster.py

```diff
@@ -10,12 +10,28 @@
 _save_cursor = None
 
 
+def tput(args):
```

Comment:
how about:
```python
def tput(*args):
    cmd = ["tput"] + args
    p = subprocess.Popen(cmd, ...)
```

Reply:
Done. Although, `*args` is a tuple so we can't concatenate.